### ISSUE TITLE
Performance (regression) tests

### ISSUE THREAD

**Issue by JohannesLichtenberger** [sirixdb/sirix#522]
We have to add performance regression tests.

Furthermore, we at first have to import and query large JSON files. It's not clear to me how usually these tests are automatically run as Github LSF might not even store really big files.

**Comment by mosheduminer** [comment]
It might be worth generating them (in a determistic fashion) before the tests, rather than storing them.

**Comment by JohannesLichtenberger** [comment]
They shouldn't fit in main memory, however, so they have to be stored at least directly in SirixDB

**Comment by mosheduminer** [comment]
Right, but it should be possible to generate the file anyway though, right? Similar to xml/json stream processing.

**Comment by JohannesLichtenberger** [comment]
Good point, we'd have generated deterministic data instead of real-world huge datasets, but maybe that's okay :-)
